pkg/metrics/metricsconfig.go

```diff
@@ -17,7 +17,9 @@ var (
 )
 
 type MetricsConfig struct {
-	DisabledMetrics []string `json:"disabledMetrics"`
+	DisabledMetrics    []string        `json:"disabledMetrics"`
+	UseLabelsWhitelist bool            `json:"useLabelsWhitelist,omitempty"`
+	LabelsWhitelist    map[string]bool `json:"labelsWhiteList,omitempty"`
```

Comment:
Where does this get configured? How would it be used?

Reply:
This is passed in a metricsConfigs JSON file https://github.com/kubecost/cost-analyzer-helm-chart/blob/16f11a314f43bb604a5d99bf3c92801c8a374b96/cost-analyzer/values.yaml#L3290

https://github.com/kubecost/cost-analyzer-helm-chart/blob/16f11a314f43bb604a5d99bf3c92801c8a374b96/cost-analyzer/templates/cost-analyzer-metrics-config-map-template.yaml#L11

Reply:
Would it be helpful to be explicit in the variable name that these only apply to pod labels?